apps/next-evans/lib/types.ts

```diff
@@ -1,35 +1,37 @@
+import type { EditionType } from '@friends-library/types';
 import type { gender as Gender } from '@prisma/client';
 import type { ArrowRightIcon } from '@heroicons/react/24/outline';
 
+export { type EditionType };
 export type HeroIcon = typeof ArrowRightIcon;
 
 export interface Friend {
+  id: UUID;
   name: string;
   slug: string;
-  id: string;
   gender: 'male' | 'female' | 'mixed';
   description: string;
   quotes: Array<{ quote: string; cite: string }>;
   born: number | null;
   died: number | null;
-  dateAdded: string;
+  createdAt: ISODateString;
   residences: Residence[];
-  documents: DocumentType[];
+  documents: FriendDocument[];
 }
 
 export interface Edition {
   type: EditionType;
   numPages: number[];
   size: 's' | 'm' | 'xl' | 'xlCondensed';
   audiobook: Audiobook | null;
-  impressionCreatedAt: string;
+  impressionCreatedAt: ISODateString;
 }
 
-export interface DocumentType {
+interface FriendDocument {
```

Comment:
this old `DocumentType` was only ever being used on the `Friend` type, so i stopped exporting it, and that allowed me to call the other, ubiquitous type `Document`.